content/developer/reference/frontend/odoo_editor.rst

```diff
@@ -0,0 +1,120 @@
+
+.. highlight:: javascript
+
+.. default-domain:: js
+
+===========
+Odoo Editor
+===========
+
+Odoo Editor is Odoo's own rich text editor. It has replaced the Summernote
+library since Odoo 15.0. It can be found at
+*web_editor/static/src/js/editor/odoo-editor*.
+
+Full documentation will come soon.
+
+Tests
+=====
+
+The editor comes with its own series of unit tests that can be seen at the route
+*/web_editor/tests*.
```

Comment:
I'm not too sure about mentioning this. Does it add any value to the page, considering that most Odoo modules have unit tests too?

Reply:
This is the only module that has a separate page for its unit tests. Other unit tests are at `/web/tests` but the editor has its tests at `/web_editor/tests` and (unlike all the others) doesn't use QUnit.

Reply:
Oh ok, then it should probably be mentioned since it's the interesting part of the information ;)